packages/cms-lib/lib/config.js

```diff
@@ -297,7 +317,11 @@ const updatePortalConfig = configOptions => {
       config.portals = [nextPortalConfig];
     }
   }
-  writeConfig();
+
+  // TODO - Implement a better solution for decoupling this
+  if (!nextPortalConfig.inMemoryOnly) {
```

Comment:
Perhaps we just follow the single responsibility principle. So we'd have update and write functions that we can call in succession in our command code.